### ISSUE TITLE
scheduler blank card 

### ISSUE THREAD

**Issue by elRadix** [nielsfaber/scheduler-card#212]
Using the current latest version, I still have blank page when adding a new card

```
scheduler-card.js:286 Uncaught (in promise) TypeError: Cannot read property 'attributes' of undefined
    at Ii (scheduler-card.js:286)
    at scheduler-card.js:286
    at Array.map (<anonymous>)
    at Ii (scheduler-card.js:286)
    at Ri (scheduler-card.js:286)
    at Bi (scheduler-card.js:286)
    at scheduler-card.js:445
    at Array.filter (<anonymous>)
    at HTMLElement.getGroups (scheduler-card.js:445)
    at HTMLElement.render (scheduler-card.js:445)
Ii @ scheduler-card.js:286
(anonymous) @ scheduler-card.js:286
Ii @ scheduler-card.js:286
Ri @ scheduler-card.js:286
Bi @ scheduler-card.js:286
(anonymous) @ scheduler-card.js:445
getGroups @ scheduler-card.js:445
render @ scheduler-card.js:445
update @ scheduler-card.js:187
performUpdate @ scheduler-card.js:148
_enqueueUpdate @ scheduler-card.js:148
async function (async)
_enqueueUpdate @ scheduler-card.js:148
requestUpdateInternal @ scheduler-card.js:148
initialize @ scheduler-card.js:148
initialize @ scheduler-card.js:187
F @ scheduler-card.js:148
se @ scheduler-card.js:187
os @ scheduler-card.js:445
_clone @ scheduler-card.js:56
__commitTemplateResult @ scheduler-card.js:82
commit @ scheduler-card.js:82
(anonymous) @ scheduler-card.js:187
se.render @ scheduler-card.js:187
update @ scheduler-card.js:187
performUpdate @ scheduler-card.js:148
_enqueueUpdate @ scheduler-card.js:148
async function (async)
_enqueueUpdate @ scheduler-card.js:148
requestUpdateInternal @ scheduler-card.js:148
set @ scheduler-card.js:148
_addItemClick @ scheduler-card.js:1536
handleEvent @ scheduler-card.js:82
A.__boundHandleEvent @ scheduler-card.js:82
newItemClick @ scheduler-card.js:404
handleEvent @ scheduler-card.js:82
A.__boundHandleEvent @ scheduler-card.js:82
```

**Comment by nielsfaber** [comment]
> Using the current latest version, I still have blank page when adding a new card

Since you say _still_, is this related to another open issue?
I am not aware of this problem.

It seems caused by one of your entities in HA, which the card is not compatible with.
Since the card is compressed code, the best approach would be if you can try installing an uncompressed version of the card.

In that case the error report will be more informative, and I can figure out where things go wrong.

**Comment by elRadix** [comment]
yes I have this issue at least 3 version ago, hoping it would be fixed during your updates.
how can install the uncompressed version?

this is my config and json 
```
- type: custom:scheduler-card
  discover_existing: true
  standard_configuration: true
  time_step: 1 
  include:
     - input_boolean
     - automation
     - media_player
     - climate
     - switch
     - light
     - group
```

`{"id": "bhgfzulhgvoirrskdaty", "entity_id": [{"entity_id": "switch.stp_eetkamer_tv", "domain": "switch", "friendly_name": "samsung"}], "friendly_name": "Samsung TV", "enable": "true", "enable_1": "true", "enable_2": "true", "enable_3": "true", "enable_4": "true", "enable_5": "true", "enable_6": "true", "enable_7": "true", "ON_1": "", "ON_2": "", "ON_3": "", "ON_4": "", "ON_5": "", "ON_6": "", "ON_7": "", "OFF_1": "08:15", "OFF_2": "08:15", "OFF_3": "08:15", "OFF_4": "08:15", "OFF_5": "08:15", "OFF_6": "", "OFF_7": ""}`

**Comment by nielsfaber** [comment]
Could you try the attached version?
It won't fix anything, but the error message in the browser should give better insight.

[scheduler-card.zip](https://github.com/nielsfaber/scheduler-card/files/5540956/scheduler-card.zip)

**Comment by elRadix** [comment]
hope this helps

![image](https://user-images.githubusercontent.com/6258894/99150700-a3ccf400-2696-11eb-8fe7-5bc5f3900279.png)

**Comment by KTibow** [comment]
That isn't helping. Make sure you delete the `gz` file and clear your cache, because it isn't any different from before.

**Comment by elRadix** [comment]
remove the file as suggested 

```
scheduler-card.js:6826 Uncaught (in promise) TypeError: Cannot read property 'attributes' of undefined
    at standardActions (scheduler-card.js:6826)
    at scheduler-card.js:6827
    at Array.map (<anonymous>)
    at standardActions (scheduler-card.js:6827)
    at computeEntityActionConfig (scheduler-card.js:6906)
    at computeEntityActions (scheduler-card.js:6991)
    at scheduler-card.js:7893
    at Array.filter (<anonymous>)
    at HTMLElement.getGroups (scheduler-card.js:7893)
    at HTMLElement.render (scheduler-card.js:7917)
```

**Comment by KTibow** [comment]
@nielsfaber do you know what this means?

**Comment by nielsfaber** [comment]
The card fails to load the configuration of one of the `group` entities in your HA config.

First all entities from your HA config are filtered, to get the list of entities that are in `include`.
For these entities the actions are calculated.
Which happens in `getGroups`:
```
    const entities = Object.keys(this.hass.states)
      .filter(e => entityFilter(e, this.config!))
      .filter(e => computeEntityActions(e, this.hass!, this.config!).length);
```
The `computeEntityActions` (line 6991) calls:
```
actionList = standardActions(entity, hass, true);
```
Where `entity == e` (the entity ID).

In standardActions:
```
  const stateObj = hass.states[entity_id];
```
Where `entity_id == entity` (still the same entity ID).

In line 6826:
```
    case 'group':
      const entities: string[] = stateObj.attributes.entity_id! || [];
```

And the error suggests that `stateObj` is `undefined`.


So long story short: at instance 1 the list of entity_IDs from HA is taken, at instance 2, one of these entity_IDs does not exist in HA anymore.
No idea what is wrong.

@elRadix Do you have a `group` entity in HA which is broken/restored or anything?

**Comment by KTibow** [comment]
I think the quickest way to fix this would be to delete it from the UI, and re-add it.

**Comment by elRadix** [comment]
I have indeed group in my card config, I have put in comment and now its working.

This means that the group feature is not working as intended?

How to use the group feature? I have some  switches, light or device trakcers (to be used  condidtion home ) in groups defined 

```
- type: custom:scheduler-card
  discover_existing: true
  standard_configuration: true
  time_step: 1
  include:
    - input_boolean
    - media_player
    - climate
    - switch
    - light
 #  - groups:
```

**Comment by nielsfaber** [comment]
No nothing is wrong with your usage, something is wrong in the card.
Could you try with this version maybe?
This one will spam your browser a bit, but it will give more info about the entity and the point of failure.

[scheduler-card.zip](https://github.com/nielsfaber/scheduler-card/files/5541097/scheduler-card.zip)

**Comment by nielsfaber** [comment]
Any update on this?
